### ISSUE TITLE
Blink after changing automatic setting

### ISSUE THREAD

**Issue by Nico207** [raatmarien/red-moon#19]
Hi, 
I have a problem when I change the automatic setting : Red Moon turn completely crazy (on, off, on, off,...) 

My config :
Cyanogemod 11
Red Moon 2.6.1

Red Moon config :
Lower brightness checked
Automatically turn on filter on custom times (on:22h, off:6h)
Keep running after reboot : unchecked
Always open on startup : checked

Step to do that:
Red Moon is on and it's actually after "on time" (22h in my case) 
Go into settings and change "custom times" into sun to enter disco mode. 

I have another question :
What is the difference between "Keep running after reboot" and "Always open on startup" 

Thanks a lot for this great application !

**Comment by raatmarien** [comment]
Hey, thanks for letting me know.

It turns out I misinterpreted what a function of Java.util.Calendar does, which makes the scheduling feature misfunction on the last day of the month. See #18.

The scheduling mode should work as expected again on the first of April and I've released a new version (v2.6.2) that fixes this bug. This new version should be available on f-droid in a day or two. 

Sorry for the inconvenience.
